### PR TITLE
Set standard name for Cancel button, to align translations

### PR DESCRIPTION
**Pull request by CarlosGS**
## Proposed change

From a brief discussion in "Lokalise->Polymer frontend" for `ui::panel::lovelace::editor::save_config::cancel`

Screenshot_20210414_092008

The original english text for the buttons is **"Never mind"** and "Take control".

Desired action for "Never mind" is to Cancel and close the prompt, however:
- Spanish translation was "No importa", in context "Sure, I don't mind" (!!)
- Default french translation was cheesy: "Oublie ce que j'ai dit, c'est pas grave" / "Forget what I just said, it is not important" :smile: 

From Dannie Rothman at Lokalise:

> I Agree, Danish, Swedish and Norwegian also ends up as something like "Forget it" or "Not really" - i changed the Danish version to something matching "Cancel" 

Proposed PR change is "Never mind" becoming "Cancel" to make it more intuitive and standard UI, aligned across languages.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @CarlosGS, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;frontend#8914).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by CarlosGS** [comment]
More examples of the issue:

Yesterday I changed the spanish translation to "Mejor no", which is the closest expression I could find to accurately replace "Never mind" in the same context. But the edit has been reverted to "No importa" without realising of the context. This confusion can easily happen with any language, hence the suggestion to use "Cancel".

Latin spanish uses "Olvídalo" / "Forget about it", which is OK, but still feels a bit orthopedic.

**Review by spacegaier — APPROVED** [review]
Valid point 👍

File: src/translations/en.json [hunk 1]
```diff
@@ -2821,7 +2821,7 @@
             "yaml_control": "To take control in YAML mode, create a YAML file with the name you specified in your config for this dashboard, or the default 'ui-lovelace.yaml'.",
             "yaml_config": "To help you start here is the current config of this dashboard:",
             "empty_config": "Start with an empty dashboard",
-            "cancel": "Never mind",
+            "cancel": "Cancel",
```
Comment (bramkragten) [inline src/translations/en.json:2824]: We have common translation for cancel: `ui.common.cancel`

**Comment by CarlosGS** [comment]
There you go, thanks for the tip @bramkragten! :)

PS: Sorry for the PR close/reopen noise.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**